src/promptflow-devkit/promptflow/_sdk/_utils.py

```diff
@@ -1028,6 +1030,22 @@ def callable_to_entry_string(callable_obj: Callable) -> str:
     return f"{module_str}:{func_str}"
 
 
+def is_flex_run(run: "Run") -> bool:
+    if run._run_source == RunInfoSources.LOCAL:
+        try:
+            # The flow yaml may have been temporarily generated and deleted after creating a run.
+            # So check_flow_exist=False
+            return is_flex_flow(flow_path=run.flow, check_flow_exist=False)
+        except Exception as e:
+            # For run with incomplete flow snapshot, ignore load flow error to make sure it can still show.
+            logger.debug(f"Failed to check is flex flow from {run.flow} due to {e}.")
+            return False
+    elif run._run_source in [RunInfoSources.INDEX_SERVICE, RunInfoSources.RUN_HISTORY]:
+        return run._properties.get("azureml.promptflow.run_mode") == "Eager"
```

Comment:
make "Eager“  an constant? change it to Flex?

Reply:
Done